<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="MatrixWidget" name="matrixWidget" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="linesArgsLabel">
          <property name="text">
           <string>Введите количество строк</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="linesArgslineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="columsArgsLabel">
          <property name="text">
           <string>Введите количество столбцов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="columnsArgslineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="answerButton">
          <property name="text">
           <string>Перейти</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatrixWidget</class>
   <extends>QWidget</extends>
   <header>matrixwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
